### PR TITLE
Fix script error

### PR DESCRIPTION
**Pull request by ichung08**
## Description

Bug bash bug fixes

## 📋 Checklist

- [ ] Added/updated applicable tests
- [ ] Added/updated examples in the `examples/` directory
- [ ] Updated any related [documentation](https://github.com/astronomer/docs/)

### CODE REVIEW

File: .github/workflows/build-release-import-script.yml [hunk 1]
```diff
@@ -36,30 +41,31 @@ jobs:
       - run: make validate-fmt
 
   goreleaser:
+    needs: [build, lint]
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29 # v4.1.6
```
Comment (vandyliu) [inline .github/workflows/build-release-import-script.yml:41]: Any reason why we're switching the version?

File: .goreleaser.import.yml [hunk 2]
```diff
@@ -10,11 +10,15 @@ builds:
     goarch:
       - amd64
       - arm64
+    env:
+      - CGO_ENABLED=0
+    ldflags:
+      - '-s -w -X main.version={{.Version}} -X main.commit={{.Commit}}'
```
Comment (vandyliu) [inline .goreleaser.import.yml:16]: What do these do?

**Review by vandyliu — APPROVED** [review]
Some nonblocking questions

File: .goreleaser.import.yml (hunk 2 above)
Comment (ichung08) [reply .goreleaser.import.yml:16]: `CGO_ENABLED=0` disables the Go interface to c libraries

`-s`: Omits the symbol table and debug information.
`-w`: Omits the DWARF symbol table.

the other ones inject the version and commit - i removed as we don't need that here

File: .github/workflows/build-release-import-script.yml (hunk 1 above)
Comment (ichung08) [reply .github/workflows/build-release-import-script.yml:41]: nope, changed

**Pull request merged**